co2eq/flights/index.js

```diff
@@ -246,10 +252,12 @@ export function activityDistance(activity) {
 export function carbonEmissions(activity) {
   const distance = activityDistance(activity);
   const [passengerLoadFactor, passengerToFreightRatio] = getLoadFactors(activity);
-
   if (!Number.isFinite(distance)) {
     throw new Error(`Incorrect distance obtained: ${distance}`);
   }
+  if (!Number.isFinite(passengerLoadFactor)) {
+    throw new Error(`Incorrect load factor obtained: ${passengerLoadFactor}`);
+  }
```

Comment:
Why not check the freight ratio as well?

It should be a function that returns a number from a boolean. Optionally, we could check that it is indeed a ratio by verifying that it is between 0 and 1

Reply:
Good idea. However, I'm not sure I see an easy way to test the function here, as we don't (yet) know the argument to that function?
I looked at the code and couldn't find anyplace else where it could be added.

Reply:
what about?
```
if (!Number.isFinite(passengerFreightRatio(true)) |  !Number.isFinite(passengerFreightRatio(false))) {
    throw new Error(`Incorrect load factor obtained: ${passengerLoadFactor}`);
  }
```

to at least check if they are numbers, not sure about checking they are between 0 and 1 though. 

If this check does not work, I'll just accept the PR

Reply:
good idea, let me add that